stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,200 @@
+
+## Responsiveness
+
+The action sheet is fully responsive. On smartphones, the actions are displayed as a list inside a dialog.
+On tablets and desktop devices, the actions are displayed in a popover.
+        `,
+        tags: ['a11y', 'f3', 'theme']
+    }
+};
+
+export const actionSheetDesktop = () => `
+<div class="fd-popover">
+    <div class="fd-popover__control">
+        <button class="fd-button fd-button--compact fd-button--transparent" 
+                aria-label="More Options" aria-controls="actionSheetDesktop" aria-expanded="false" 
+                aria-haspopup="true" onclick="onPopoverClick('actionSheetDesktop');">
+            <i class="sap-icon--settings"></i>
+        </button>
+    </div>
+    <div class="fd-popover__body" aria-hidden="true" id="actionSheetDesktop">
+        <ul class="fd-action-sheet fd-action-sheet--compact" role="list" aria-label="List of contextual options">
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--attachment"></i>
+                    <span class="fd-button__text">Upload file</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--add"></i>
+                    <span class="fd-button__text">Assign user</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--print"></i>
+                    <span class="fd-button__text">Print</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left">
+                    <i class="sap-icon--download"></i>
+                    <span class="fd-button__text">Download</span>
+                </button>
+            </li>
+            <li class="fd-action-sheet__item" role="listitem">
+                <button class="fd-button fd-button--full-width fd-button--compact fd-button--transparent fd-button--text-alignment-left"
+                        onclick="onPopoverClick('actionSheetDesktop');">
+                    <span class="fd-button__text">Decide Later</span>
+                </button>
+            </li>
+        </ul>
+    </div>
+</div>
+<div style="height: 300px"></div>
+`;
+
+actionSheetDesktop.parameters = {
+    docs: {
```

Comment:
IDs are only to identify popover. It's needed for method `onPopoverClick` to find proper element, that will have changed display mode. Fragment you linked is just to keep height of example's frame

Reply:
As you can see above button has linked property `onclick="onPopoverClick('actionSheetTablet');"`. The `onPopoverClick` method takes id argument, which is needed to find element by `document.getElementById(id);` In most of cases we use random 8 chars

Reply:
Got it. So what differentiates one component type to the next? (difference between Desktop and Tablet for instance?) Is it the `aria-controls`?

The reason why I ask is because we usually reference modifier classes in the docs, to give readers a quick hint on how they can accomplish a certain look/style. What would we tell our users about how to achieve the Desktop mode for this component?

Reply:
So the only difference between those 2 usages (Desktop and Tablet) is that Desktop uses compact mode for container `fd-action-sheet--compact` and for buttons `fd-button--compact`

Reply:
"The action sheet is displayed in desktop mode by adding the `fd-action-sheet--compact` modifier class to the container, and `fd-button--compact` to each button."